### ISSUE TITLE
[Scheduling Tool] Limit the number of meetings teams can schedule

### ISSUE THREAD

**Issue by ronniegeraghty** [Azure/azure-sdk-tools#4671]
We started to see that some of the teams using the scheduling tool are scheduling many meetings in advance. Some even scheduling 1 meeting every week for a full month. Often these are just place holders in case the team is ready to meet with the arch board at that time but often end up getting rescheduled or cancelled. I propose we limit the number of meetings a team can schedule to at most 2 upcoming meetings. This way teams cannot fill up the calendar with place holder meetings. 

@kyle-patterson @ladonnaq @maririos

**Comment by maririos** [comment]
Yeah definitely a behavior we have been seeing. How do you want us to manage this?
- 2 upcoming meetings for a month?
- 2 upcoming meetings in the same week?

As for the who:
- Should the filtering happen by `Service name + Product Name (not available yet in the tool but it is in the works) + Sub service name` ?
- Or is there another option/combination we should look into?

As for the experience, what would you like for it to happen?
- Error with a specific message? what message?

**Comment by ronniegeraghty** [comment]
My recommendation would be a service team can only have 2 upcoming meetings. So, if they scheduled two meetings, they wouldn't be able to schedule another until one of their meetings occurs. 

For the who, I think you have it right. The service name + product name. 

The message should say something along the lines of: "Service teams can have no more than 2 standing meetings at a time. Please wait until after one of your scheduled meetings to schedule another. If this is related to something urgent, please consider adding this topic into one of your already scheduled review sessions."

**Comment by maririos** [comment]
Break out the work for this feature into a roadmap

**Comment by maririos** [comment]
Work to do:
The trigger will be when a user selects a product. Then:
- Determine if a `service-product` has 2 `In-progress` meetings scheduled for meeting type `arch board`.
- If so:
  - Create button should be disabled
  - Have a pop-up with the message: 
  > Service teams can have no more than 2 standing meetings at a time. Please wait until after one of your scheduled meetings to schedule another. If this is related to something urgent, please consider adding this topic into one of your already scheduled review sessions.
  The pop-up should have a button that says `Ok` or something like that. when clicked, I think it will be better to go back to the List meetings screen

**Comment by maririos** [comment]
Est time: 2 - 3 days / 1 person

**Comment by maririos** [comment]
Lowering the priority
